README.md

```diff
@@ -185,330 +173,10 @@ Explore `visualize_dataset.ipynb` to know more about the dataset.
-```
-
-Currently logged informations are:
-- Train multi class accuracy
-- Train multi class jaccard index
-- Train loss
-- Validation multi class accuracy
-- Validation multi class jaccard index
-- Validation loss
-- 10 prediction results (images)
-- Learning rate
-
-## Testing
-
-Once your model has been trained, you can evaluate it on the test set specified in your datamodule. This can be done using the same configuration file used for training.
-
-```
-ftw model test --help
-
-Usage: ftw model test [OPTIONS] [CLI_ARGS]...
-
-  Test the model
-
-Options:
-  -m, --model TEXT            Path to model checkpoint  [required]
-  --dir TEXT                  Root directory of dataset
-  --gpu INTEGER               GPU to use
-  --countries TEXT            Countries to evaluate on  [required]
-  --postprocess               Apply postprocessing to the model output
-  --iou_threshold FLOAT       IoU threshold for matching predictions to ground
-                              truths
- -o, --out TEXT               Output file for metrics
-  --model_predicts_3_classes  Whether the model predicts 3 classes or 2
-                              classes
-  --test_on_3_classes         Whether to test on 3 classes or 2 classes
-  --temporal_options TEXT     Temporal option (stacked, windowA, windowB,
-                              etc.)
-  --help                      Show this message and exit.
-```
-
-
-### Test a model
-
-Using FTW cli commands to test the model, you can pass specific options, such as selecting the GPUs, providing checkpoints, specifying countries for testing, and postprocessing results:
-
-```bash
-ftw model test --gpu 0 --dir /path/to/dataset --model logs/path_to_model/checkpoints/last.ckpt --countries country_to_test_on --out results.csv
-```
-
-This will output test results into `results.csv` after running on the selected GPUs and processing the specified countries.
-
-Note: If data directory path is custom (not default ./data/) then make sure to pass custom data directory path in testing using ```--dir custom_dir/ftw```.
-
-## Parallel experimentation
-
-For running multiple experiments across different GPUs in parallel, the provided Python script `run_experiments.py` can be used. It efficiently manages and distributes training tasks across available GPUs by using multiprocessing and queuing mechanisms.
-
-### Run experiments in parallel
-
-1. Define the list of experiment configuration files in the `experiment_configs` list.
-2. Specify the list of GPUs in the `GPUS` variable (e.g., `[0,1,2,3]`).
-3. Set `DRY_RUN = False` to execute the experiments.
-
-The script automatically detects the available GPUs and runs the specified experiments on them. Each experiment will use the configuration file specified in `experiment_configs`.
-
-```bash
-python run_experiments.py
-```
-
-The script will distribute the experiments across the specified GPUs using a queue, with each experiment running on the corresponding GPU.
-
-## Inference
```

Comment:
This went a little further than I was hoping - could we move this section back out? Made #46 to track.